From 45f5b612c402856de6d84c0b617bc129d9ee2972 Mon Sep 17 00:00:00 2001
From: Jonathan Bastien-Filiatrault <joe@x2a.org>
Date: Fri, 05 Apr 2013 02:48:27 +0000
Subject: Add missing command-N corresponding to run_command-N for global keybindings.

Also, would a default value of an empty string be better than a single space ?
---
diff --git a/src/org.mate.marco.gschema.xml b/src/org.mate.marco.gschema.xml
index 03805aa..d3841f9 100644
--- a/src/org.mate.marco.gschema.xml
+++ b/src/org.mate.marco.gschema.xml
@@ -500,6 +500,61 @@
       <summary>Commands to run in response to keybindings</summary>
       <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
     </key>
+    <key name="command-2" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-3" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-4" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-5" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-6" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-7" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-8" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-9" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-10" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-11" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
+    <key name="command-12" type="s">
+      <default>' '</default>
+      <summary>Commands to run in response to keybindings</summary>
+      <description>The /apps/marco/global_keybindings/run_command_N keys define keybindings that correspond to these commands. Pressing the keybinding for run_command_N will execute command_N.</description>
+    </key>
     <key name="command-screenshot" type="s">
       <default>'mate-screenshot'</default>
       <summary>The screenshot command</summary>
--
cgit 
